starboard/shared/win32/file_close.cc

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// #if SB_API_VERSION < 16
```

Comment:
too early for deprecation, please remove the commeent.

Reply:
Removed.